modules/roomify/roomify_accommodation_booking/roomify_accommodation_booking.module

```diff
@@ -919,8 +919,9 @@ function roomify_accommodation_booking_confirmation_form($form, &$form_state, $s
     if (variable_get('roomify_accommodation_booking_allow_instant_bookings', 1)) {
       if (isset($property->field_sp_allow_instant_bookings[LANGUAGE_NONE][0]['value']) &&
           $property->field_sp_allow_instant_bookings[LANGUAGE_NONE][0]['value']) {
-        if (isset($type->field_st_allow_instant_bookings[LANGUAGE_NONE][0]['value']) &&
-            $type->field_st_allow_instant_bookings[LANGUAGE_NONE][0]['value']) {
+        // If it's a home we don't need to go over.
+        if ($type->type == 'home' || (isset($type->field_st_allow_instant_bookings[LANGUAGE_NONE][0]['value']) &&
```

Comment:
can we replace the two tests with a single !empty($type->field_st_allow_instant_bookings[LANGUAGE_NONE][0]['value']) ? Also I'm not quite sure what the comment means :)